contributing.qmd

```diff
@@ -2,63 +2,91 @@
 
-Once the pull request is opened, and the github `preview.yml` workflow runs ("Deploy PR previews"), you should have a preview available for review at https://developmentseed.org/cloud-optimized-geospatial-formats-guide/pr-preview/pr-<YOUR-PR-NUMBER-HERE>.
+### Specific Contributions
 
-### 1. Adding a new format
+#### 1. Adding a New Format
 
-1. Create a folder with the formats name and, within that folder, an `intro.qmd`. The `intro.qmd` file should describe the basics about that format.
+Follow the steps outlined in the General Steps, then:
+
+1. Create a folder with the format's name and, within that folder, an `intro.qmd`.
 2. Link to the `intro.qmd` page in the `index.qmd` (the **Welcome** page) file and `_quarto.yml` table of contents.
-3. Optionally, add a notebook with examples of creating and accessing (via the cloud) a file of that format. We suggest including an [`environment.yml` file](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#create-env-file-manually) that defines the Conda packages necessary for the given notebook.
 
-### 2. Modify or add to an existing format
+#### 2. Modify or Add to an Existing Format
 
 Feel free to modify or add to existing content if you think it could be improved.
 
-### 3. Adding a cookbook
+#### 3. Adding a Cookbook
+
+Cookbooks should address common questions and present solutions for cloud-optimized access and visualization. To create a cookbook, either add a notebook directly to this repository in the cookbooks directory OR use an external link and add it to cookbooks/index.qmd.
+
+#### 4. (Optional) Update Slides
+
+If you have made substantive changes, consider updating the [Overview Slides](./overview.qmd). These slides are generated using [Quarto and Reveal.js](https://quarto.org/docs/presentations/revealjs/) so can be updated with markdown syntax.
+
+#### 5. Add Yourself to the List of Authors
+
+Add yourself to the list of authors on the [Welcome](./index.qmd#authors) page.
+
+#### 6. Final Steps Before Merging
 
-Cookbooks should address common questions and present solutions for cloud-optimized access and visualization. For example:
+Once your PR is approved and all checks have passed, a project maintainer will merge your changes into the main repository.
 
-* How do I chose the chunk shape for my Zarr?
-* How do I determine and preview the resampling algorithm for my COG overview?
-* How do I create STAC metadata for my cloud-optimized data?
-* How do I provide visualizations fo this data?
-* How do I provide subsetted/query/analytical access this data?
+## Licensing
 
-To create a cookbook, either add a notebook directly to this repository in the cookbooks directory OR use an external link and add it to cookbooks/index.qmd.
+Contributions to this project are accepted under the Creative Commons Attribution 4.0 International license.
```

Comment:
TODO: figure out if people are assigning copyright to DevSeed or Radiant (so that changes can be made later without needing author permission - e.g. upgrading to a newer CC version later)

Reply:
Opened an issue: https://github.com/developmentseed/cloud-optimized-geospatial-formats-guide/issues/52

Reply:
not a blocker for this PR, but is a blocker to accepting PRs from outside authors.